test/components/rankers/test_transformers_similarity.py

```diff
@@ -421,20 +423,21 @@ def test_run_small_batch_size(self, query, docs_before_texts, expected_first_tex
 
     #  Returns an empty list if no documents are provided
     @pytest.mark.integration
+    @pytest.mark.slow
```

Comment:
@anakin87 could be out of scope for this PR, but this test could be made into a unit test with a patch on the `warm_up` method. 

Maybe worth noting down somewhere which of these tests could be made into unit tests?

Reply:
done in https://github.com/deepset-ai/haystack/pull/9296/commits/a31d1574eeede2ea202b44b66e65f09af454e4f9